tools/install.py

```diff
@@ -15,32 +16,33 @@
 def register(command):
     command.help = "Install the system onto the target system"
 
-    # Configuration
-    command.add_argument("target", help="the target host to install the packages to")
+    command.add_argument("target", help="The target host or directory to install the packages to")
 
-    command.add_argument("--user", "-u", help="the user to install onto the target with")
+    command.add_argument("--local", "-l", dest="local", action="store_true", help="Install to a local directory")
+
+    command.add_argument("--user", "-u", help="The user to install onto the target with")
 
     command.add_argument(
         "--config",
         "-c",
         choices=["new", "n", "update", "u", "overwrite", "o", "ignore", "i"],
         default="new",
-        help="method to use for configuration files when installing",
+        help="The method to use for configuration files when installing",
     )
 
     command.add_argument("-t", "--toolchain", dest="toolchain", action="store_true")
```

Comment:
should probably add a default to this too
```suggestion
    command.add_argument("-t", "--toolchain", dest="toolchain", default=False, action="store_true")
```